src/exchange/wizard-hosted-creation/email-creation/add/add.html

```diff
@@ -0,0 +1,170 @@
+                           data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_emailCreation_add_password_error_exchangePreferences')">
+                        </p>
+                    </div>
+                </div>
+
+                <div class="form-group"
+                     data-ng-class="{ 'has-error': $ctrl.addForm.password.$dirty && $ctrl.addForm.passwordConfirmation.$dirty && $ctrl.addForm.passwordConfirmation.$invalid }">
+                    <label class="col-md-4 control-label required"
+                           for="passwordConfirmation"
+                           data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_emailCreation_add_passwordConfirmation_header')"></label>
+                    <div class="col-md-8">
+                        <input type="password"
+                               class="form-control"
+                               id="passwordConfirmation"
+                               name="passwordConfirmation"
+                               required
+                               maxlength="256"
+                               data-ng-model="$ctrl.model.passwordConfirmation"
+                               data-ng-change="$ctrl.validatePassword()">
+                        <p class="help-block"
+                           data-ng-if="$ctrl.addForm.password.$dirty && $ctrl.addForm.passwordConfirmation.$dirty && $ctrl.addForm.passwordConfirmation.$error.required"
+                           data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_emailCreation_add_login_required')">
+                        </p>
+                    </div>
+                </div>
+
+                <p class="required" data-i18n-static="exchange_ACTION_add_account_step1_mandatory_fields"></p>
+            </form>
+        </div>
+
+        <div data-wizard-step>
+            <dl class="dl-horizontal dl-lg">
+                <dt data-i18n-static="exchange_ACTION_add_account_step2_email_label"></dt>
+                <dd data-ng-bind-template="{{$ctrl.model.login.toLowerCase()}}@{{$ctrl.model.domain}}" class="word-break"></dd>
+
+                <dt data-ng-if="$ctrl.model.firstName" data-i18n-static="exchange_ACTION_add_account_step2_first_name_label"></dt>
+                <dd data-ng-if="$ctrl.model.firstName" data-ng-bind="$ctrl.model.firstName" class="word-break"></dd>
+
+                <dt data-ng-if="$ctrl.model.lastName" data-i18n-static="exchange_ACTION_add_account_step2_last_name_label"></dt>
+                <dd data-ng-if="$ctrl.model.lastName" data-ng-bind="$ctrl.model.lastName" class="word-break"></dd>
+
+                <dt data-ng-if="$ctrl.model.displayName" data-i18n-static="exchange_ACTION_add_account_step2_display_name_label"></dt>
+                <dd data-ng-if="$ctrl.model.displayName" data-ng-bind="$ctrl.model.displayName" class="word-break"></dd>
+            </dl>
+
+            <form class="mt-3" data-ng-if="$ctrl.canBeMigrated">
```

Comment:
Apply your `data-ng-if="$ctrl.canBeMigrated"` directly on the `<div class="alert alert-info">` below.